factory/mock/builtInFunctionsContainerMock.go

```diff
@@ -0,0 +1,47 @@
+package mock
```

Comment:
1. Please move this in testscommon
2. In testscommon there is already an error called ErrNotImplemented, so we should use that one
3. File is called mock, but struct is called stub

Reply:
moved